.github/workflows/xep-validation.yml

```
@@ -0,0 +1,35 @@
+name: XEP validation
+
+on:
+  pull_request:
+    branches:
+      - master
+
```

Might be helpful, so this action can be run on old PRs.

```suggestion
  # Allow to run this workflow manually from the Actions tab
  workflow_dispatch:
```

---

I'm happy to accept that suggestion if you tested that the workflow remains functional for the original intended target: pull requests.

I am reasonably sure that the original worked for PRs, as that's what I tested. I don't want to spend the time to re-test this though. :) #lazy

---

I tested `workflow_dispatch:` in [my fork](https://github.com/wurstsalat3000/) and it continues to work with pull requests. It additionally allows to manually run this workflow on any branch (but only branches from https://github.com/xsf/xeps/ , which isn't what I originally intended with my suggestion, since PRs are based on forks most of the time). Feel free to ignore :)